### DISCUSSION TITLE
[MINIGAME] Corn Maze Repurposed as Tag Minigame

### DISCUSSION THREAD

**Discussion by HomerN2763** [sunflower-land/sunflower-land#3006] [Ideas]
My idea is that it would be 1 Chaser and 10 Players. 

The tagger will be chosen randomly, but anyone with a Crumple Crown will be immediately chosen for the first 2 plays each day (if 2 or more people have the Crown than the tagger will be chosen randomly between those who do). 

The gameplay for the Player is no different from the Corn Maze game currently: You have 3 hearts, Common enemies to dodge, but this time around the Chaser will chase after you, and if the Chaser catches you, you lose all hearts! In order to win, you must survive until the timer runs out.

As the Chaser, you catch Players before the timer runs out, or when you caught all Players. NPC enemies won't hurt you.

Sometimes crops (from the Goblin Community Treasury, probably) will be scavenged along the way for anyone to pick up.

Every participant will each pay a certain amount of SFL to play a match.
I also have a Rewards Distributions system in mind. Let's say all 11 participants each pay 1 SFL to play. The rewards will be distributed as following:

The Chaser earns 0.4 SFL per Player caught. Catching the last Player grants 2 SFL bonus.

All 10 Players surviving: 1.05 SFL each
9 Players surviving: 1.1 SFL each
8 Players surviving: 1.15 SFL each
7 Players surviving: 1.2 SFL each
6 Players surviving: 1.3 SFL each
5 Players surviving: 1.4 SFL each
4 Players surviving: 1.5 SFL each
3 Players surviving: 1.75 SFL each
2 Players surviving: 2 SFL each
1 Player surviving: 3 SFL

The Leftover SFL after the match will be burned.

**Comment by rodolfoddias** [comment]
This is an amazing idea. I see a lot of potential on it. I also see a ranking system, where you start as "wood tier" with 1 SFL bets and can reach "gold tier" with 5 SFL bets for example. Higher risks, but better rewards.

Would need to think about how to reward the team. Maybe adding a block buck fee to play a number of matches or maybe adding seasonal items to incentivize season banner purchases. Crumple Crown would be the first "gameplay utility SFT" and new kind of "masks" could be released as different skins of "Crumple Crown".

**Reply by HomerN2763** [reply]
The Ranking system seems a bit unnecessary, since it's more or less just a minigame. Players should be open to all stakes.

**Reply by HomerN2763** [reply]
Some of the leftover SFL can be rewarded to the team while most of the rest will be burnt.

**Comment by TheDcol** [comment]
Does the chaser run faster than others?

**Reply by HomerN2763** [reply]
I haven't decided about it, but personally I think Chasers would run at the same speed as the others, as being immune to the NPC Enemies is already a considerable advantage to the Chaser.

**Comment by johannesunana** [comment]
This looks like an interesting idea. I'm half imagining it, but if possible, maybe someone can help you make a visual mockup so other community developers and other players can see how it would probably be done.

**Reply by rasheed19871** [reply]
That's interesting idea hopefully the tem convert it in reality
